### ISSUE TITLE
[QUESTION] Enemy Spawn Range default values

### ISSUE THREAD

**Issue by Instaplayer** [TheAnsuz/Lethal-Company-Configurable-Company-API#23]
By default the mod sets all Enemy Spawn Ranges in Global Enemy Spawning as values between 0 and 10
![image](https://github.com/TheAnsuz/Lethal-Company-Configurable-Company-API/assets/94839182/ec61858f-92c0-46f0-b55f-f80156985796)

However according to Lethal Company wikis (both mirahaze and fandom) the actual vanilla clamps are between between 0 and 20 or `minEnemiesToSpawn` and 20 or `minOutsideEnemiesToSpawn` and 20

lethal.miraheze.org/wiki/Mechanics
https://lethal-company.fandom.com/wiki/Mechanics

For context I am running V50 of the game while both wikis are up-to-date with previous versions

Are the wikis wrong and the vanilla clamps were reduced? 
Am I mistaking the values for different ones?
Or are the default values actually reducing the maximum amount of enemies spawned at the same time?

**Comment by TheAnsuz** [comment]
Hello @Instaplayer 

(From my understanding) While `minEnemiesToSpawn` and `minOutsideEnemiesToSpawn` are there in the game, they are only used for the next *round* of enemies and only increase after certain events happen such as **grabbing the apparatus** or **landing on an eclipsed planet**.

The values you mention are not related to those, this ones are meant as a **forced minimum and maximum** spawns per hour to guarantee at least and at most the configured amount of enemies to spawn.

The values in those settings are set from 0 to 10 as there is no vanilla counterpart for this behavior. Vanilla spawns, even if set to 20 would still be limited by the amount of vents on the level, which is usually around 5-7 no matter the map size (they are not limited outside the facility).
These settings will also work even if there are no more vents in the level (by creating new ones).

I can't be sure about what the wiki says nor what is all the intrincate behavior of the game but in none of my tests I found those settings related or needed to be capped as they are capped by the phisical limit of the level.

TL;DR
I don't know if the wikies are wrong, I never found those settings to be any relevant except for certain occasions where not even the limit would matter.
Those values are not related to vanilla, the values from vanilla are not reliable for the usage of those settings, these configs clamp the amount of enemies **each hour**.
I never encountered any occasion where the amount of enemies per hour was greater than 10 (never reached 10)

Feel free to correct any information or ask anything you need!

**Comment by Instaplayer** [comment]
Ah alright. Thank you for clarification. 
I didn't know how exactly the setting functioned which is why I was confused.
